src/components/apps/utils.js

```diff
@@ -51,6 +51,21 @@ export const getAppLaunchPath = (systemId, appId, versionId) =>
 export const getSavedLaunchPath = (systemId, appId, launchId) =>
     `/${NavigationConstants.APPS}/${systemId}/${appId}/launch?saved-launch-id=${launchId}`;
 
+/**
+ * Builds a path to the direct instant launch page.
+ *
+ * @param {string} ilId The Instant Launch ID
+ * @param {string} resourcePath A file or folder path to pass to the instant launch (optional)
+ */
+export const getInstantLaunchPath = (ilId, resourcePath = "") => {
+    var uri = `/${NavigationConstants.INSTANT_LAUNCH}/${ilId}`;
+    if (resourcePath !== "") {
```

Comment:
It might be safer to just check truthiness?
```suggestion
    if (resourcePath) {
```

Reply:
Hm, that's probably fair. I can't imagine it somehow gets passed something that's a non-string and I guess it'd figure out how to interpolate it (not that it'd be a meaningful value anyway) if it did.

Reply:
I think it's happened at least once before where the UI sent `null` in a param like this to the backend 😖

Reply:
Yeah. In this case it's just generating a URL, I guess, so it'd just end up looking weird and presumably fail to load the path when trying to launch from it. Which isn't great, but not "shoving bad data into the database" bad or anything, heh.